litmus/director/openebs-self-install/run_litmus_test.yml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: openebs-self-install-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: openebs-self-install-check-litmus
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      volumes:
+      - name: secret-volume
+        secret:
+          secretName: director-user-pass
+      containers:
+      - name: ansibletest
+        image: mayadataio/dop-validator:ci
```

Comment:
why dont we use a proper release tag?

Reply:
We have used `ci` tag in all other tests. 
should we use proper release tag here `i.e 1.8.0`? @harshshekhar15

Reply:
@AmitKumarDas this image gets contains all the test cases that are present in this repo. It gets updated with the latest test cases after every PR merge in this repo. This image continuously gets updated on every merge so I am not so sure if we should use a release tag here.

Reply:
okay done @harshshekhar15